backends/tfhe-cuda-backend/cuda/src/polynomial/parameters.cuh

```diff
@@ -1,7 +1,7 @@
 #ifndef CUDA_PARAMETERS_CUH
 #define CUDA_PARAMETERS_CUH
 
-constexpr int log2(int n) { return (n <= 2) ? 1 : 1 + log2(n / 2); }
+#define log2_int(X) ((unsigned) (8*sizeof (unsigned long long) - __builtin_clzll((X)) - 1))
```

Comment:
Only works for 8 bytes integers, could be a templated function

Reply:
Probably 32 and 64 bits are enough for now.